dotcom-rendering/src/lib/discussion.ts

```diff
@@ -75,7 +74,7 @@ export interface UserProfile {
 }
 
 const baseCommentSchema = object({
-	id: number(),
+	id: transform(union([number(), string()]), (id) => id.toString()),
```

Comment:
We’re allowing either a `number` or a `string` so as to keep the diff minimal in the tests which have `createComment` and `createReply` helpers. This also makes it more agnostic to what the API sends.